### ISSUE TITLE
Migrating data from Labdoo-1.0 to Labdoo-2.0

### ISSUE THREAD

**Issue by dashohoxha** [Labdoo/Labdoo#13]
If the structure of labdoo objects is finished, then we should think about how to migrate the data from the old application to the new one. I think that the data that needs to be migrated are:
  1. Drupal users.
  2. Labdoo objects (laptops, hubs, edoovillages, dootrips, etc.)
  3. Pages and other content.

Especially the relations between the labdoo objects and also the relations with users can be a bit tricky. But I hope that there are tools that can help for making it possible.

I think that the migration should be implemented as a script (or set of scripts), that can be run at any time to perform the migration; it should not be a manual migration that can be done only once.

**Comment by dashohoxha** [comment]
Since the new version of the website is online (https://www.labdoo.org/) it seems that data migration and everything related is already done.

By the way, I like the new version of the site. It seems nice and clean to me, and it is also responsive (looks reasonably good on small screens like tablets, smartphones, etc.)

I have some small suggestions though, and I will create new issues about them.
